components/match2/wikis/leagueoflegends/big_match.lua

```diff
@@ -184,7 +184,10 @@ function BigMatch.run(frame)
 		if type(self) == 'table' then
 			champion = self.champion
 		end
```

Comment:
```suggestion
		if type(champion ) == 'table' then
			champion = self.champion
		end
		---@cast -table
```
Not sure the cast is needed
This should allow to drop the as

Reply:
```suggestion
		if type(self) == 'table' then
			champion = self.champion
			---@cast champion -table
		end
```
We prefer to use this instead of @ as?

Reply:
yes